web/src/main/java/com/bazaarvoice/emodb/web/resources/sor/DataStoreResource1.java

```diff
@@ -323,10 +325,12 @@ public Table getTableMetadata(@PathParam ("table") String table) {
     public Map<String, Object> get(@PathParam ("table") String table,
                                    @PathParam ("key") String key,
                                    @QueryParam ("consistency") @DefaultValue ("STRONG") ReadConsistencyParam consistency,
-                                   @QueryParam ("debug") BooleanParam debug) {
+                                   @QueryParam ("debug") BooleanParam debug,
+                                   @ParamRequiresPermissions("sor|update|{table}") @QueryParam ("showHiddenFields") BooleanParam showHiddenFields) {
```

Comment:
correct. I decided it was cleaner not to have to copy and paste an auth stanza requiring the subject everywhere but instead keep it with the parameter specification. I think it's fine to ask unauthorized users not to specify the restricted parameter at all.

In fact, in some ways it makes more sense this way. "showHiddenFields" is an admin feature, which you don't have access to as a mere reader. Why should you be able to specify any value for that parameter when you have no access to that feature?